get-linux-source/checkout_latest_kernel.sh

```diff
@@ -26,27 +24,27 @@ if [ ! -d "${REPO_PATH}" ]; then
 	echo
 	foldable start pull_kernel_srcs "Fetching kernel sources"
 
-	mkdir -p $(dirname "${REPO_PATH}")
-	cd $(dirname "${REPO_PATH}")
+	mkdir -p "$(dirname "${REPO_PATH}")"
+	cd "$(dirname "${REPO_PATH}")"
 	# attempt to fetch desired bpf-next repo snapshot
 	if [ -n "${SNAPSHOT_URL}" ] && wget -nv ${SNAPSHOT_URL} && tar xf bpf-next-${LINUX_SHA}.tar.gz --totals ; then
-		mv bpf-next-${LINUX_SHA} $(basename ${REPO_PATH})
+		mv "bpf-next-${LINUX_SHA}" "$(basename ${REPO_PATH})"
```

Comment:
For my understanding, why is `REPO_PATH` not quoted here similar to how `$0` is at the top of the file?

Reply:
That can be quoted too. Anyway I can suggest a change which will get rid of all $(basename ...) stuff:
```
$ PATH=/var/lib/something
$ echo ${PATH%/*}        
/var/lib
```